code/run_capsule.py

```diff
@@ -38,9 +39,9 @@ def get_input_parser() -> argparse.ArgumentParser:
                               or not to find the file extension
         - `--split_files`: Whether or not to group the files into a
                            single model or to split into seperate
-        - `--max_jobs`: The number of parallel workers to output,
-                                    default is 50.
-                                    min(length of results returned in query, 50).
+        - `--tasks_per_job`: The number tasks per job, calculated 
+                             by ceiling(number of records / tasks_per_job)
```

Comment:
I see what you're getting at, but think this explanation is confusing/unnecessary